src/scylla/db.clj

```diff
@@ -268,7 +268,8 @@
     ; uninstall the existing packages.
     (when (cu/exists? repo-file)
       (let [prev-version ((re-find #"scylladb-([\d\.]+)"
-                                   (c/exec :cat repo-file)) 1)]
+                                   (str (c/exec :cat repo-file)
+                                        "scylladb-0.0")) 1)]
```

Comment:
this will concatenate the contents of the repo-file and scylladb-0.0 if the repo file exists and is not empty. While i thought we want to have a default if repo-file doesn't exist.

Reply:
Replaced it with array [destructuring](https://clojure.org/guides/destructuring)